spec/UserMiddleware.coffee

```diff
@@ -256,7 +256,7 @@ describe 'User Middleware', ->
         mock.restore()
       it 'should perform a token exchange and update user information', (done) ->
         action = 'application:url'
-        payload = "https://app.flowhub.io?code=#{code}"
+        payload = "https://app.flowhub.io?code=#{code}&state="
```

Comment:
Should have a case with this, and one without, if this is something that github does (occationally send)